jquery.flot.js

```diff
@@ -379,9 +380,10 @@ Licensed under the MIT license.
         }
 
         function setData(d) {
+            var oldseries = series;
             series = parseData(d);
             fillInSeriesOptions();
-            processData();
+            processData(oldseries);
```

Comment:
try to reuse the datapoints from the previous setData preventing increasing memory usage and GC